CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to WooCommerce Admin
+
+Hi! Thank you for your interest in contributing to WooCommerce Admin. We appreciate it.
```

Comment:
Are we sticking with `wc-admin` as a name? If not, maybe best to change this to `wc-admin` so when it gets changed this isn't overlooked.

Reply:
Nevermind... the plugin is titled WooCommerce Admin anyways 😄

Reply:
Ultimately, a lot of this sort of documentation will go away or be absorbed into WooCommerce's repo when we merge this into that plugin, so the name "WooCommerce Admin" won't really be "stuck" with long term - but for now if we say "WooCommerce Admin" - we match the rest of this repo.

Reply:
> Are we sticking with wc-admin as a name? 

The admin classes in Woo Core are already WC_Admin_* -- so we picked a future compatible name for code/classes :).